lib/config/env/production.js

```diff
@@ -4,16 +4,16 @@ module.exports = {
   env: 'production',
   mongo: {
     uri: process.env.OPENSHIFT_MONGODB_DB_URL + process.env.OPENSHIFT_APP_NAME ||
-         'mongodb://localhost/gdgx-hub-prod'
+         'mongodb://mongodb-mongo-db-1gu7/gdgx-hub-prod'
   },
   mail: {
     sender: "GDG[x] Hub <hub@gdgx.io>",
     transport: "Direct",
-    error_recipient: "maui@gdgac.org"
+    error_recipient: "splaktar@gmail.com"
   },
   redis: {
-  	host: process.env.OPENSHIFT_REDIS_DB_HOST,
-  	port: process.env.OPENSHIFT_REDIS_DB_PORT,
-  	password: process.env.OPENSHIFT_REDIS_DB_PASSWORD
+  	host: process.env.OPENSHIFT_REDIS_DB_HOST || '130.211.142.195',
```

Comment:
That's the main node in the production Redis. This could probably be done better, but I am waiting to get everything working outside of OpenShift before removing all OpenShift references. That will be a separate PR.